src/lib/Scenes/Fair/Screens/FairDetail.tsx

```diff
@@ -150,10 +151,13 @@ export class FairDetail extends React.Component<Props, State> {
         )
       case "information":
         return (
-          <>
-            <CaretButton onPress={() => data.onViewMoreInfoPressed()} text="View more information" />
-            <Separator mt={2} />
-          </>
+          shouldGoStraightToWebsite(this.props.fair) ||
```

Comment:
Can we move this logic into the `updateSections` function and apply the conditional before we push the `location` type into the sections array? The padding is still rendering around the box wrapper even if we return `null` here.

Reply:
Yep, done 👍